dragula.js

```diff
@@ -445,6 +454,56 @@ function dragula (initialContainers, options) {
     }
   }
 
+  function onMouseMove (e) {
+    var y = getCoord('pageY', e);
+    var x = getCoord('pageX', e);
+    drake.previousY = y;
+
+    if (drake.dragging) {
+      for (var i = 0; i < drake.containers.length; i++) {
+        var container = drake.containers[i];
+
+        var topTriggerContainer = {
+          top: container.offsetTop,
+          right: container.offsetWidth + container.offsetLeft,
+          bottom: container.offsetTop + o.scrollTriggerTop,
+          left: container.offsetLeft
+        };
+
+        var bottomTriggerContainer = {
+          top: container.offsetTop + container.offsetHeight - o.scrollTriggerBottom,
+          right: container.offsetWidth + container.offsetLeft,
+          bottom: container.offsetTop + container.offsetHeight,
+          left: container.offsetLeft
+        };
+
+        if (y >= topTriggerContainer.top && y <= topTriggerContainer.bottom
+        && x >= topTriggerContainer.left && x <= topTriggerContainer.right) {
+          scrollUp(container, y);
+        } else if (y >= bottomTriggerContainer.top && y <= bottomTriggerContainer.bottom
+          && x >= bottomTriggerContainer.left && x <= bottomTriggerContainer.right) {
+          scrollDown(container, y);
+        }
+      }
+    }
+  }
+
+  function scrollDown (container, y) {
+    if (y === drake.previousY) {
```

Comment:
Let's create local variables inside the `dragula` method instead of exposing them on `drake`.